### ISSUE TITLE
Can the resolution be higher? For example 1280*768

### ISSUE THREAD

**Issue by yuanyiming** [divamgupta/diffusionbee-stable-diffusion-ui#35]
Can the resolution be higher? For example 1280*768

**Comment by Kurville** [comment]
Higher would be even better. Like printing quality higher.

**Comment by fingermouse** [comment]
Quite aside from larger images requiring substantially more memory and time to generate, the Stable Diffusion model was trained on 512x512 images and is best at this size and a 1:1 aspect ratio. It can technically go larger but you won't get a lot more detail. It's still early days, though, so I'm sure higher resolution models will come along eventually (assuming you have a system with 64GB+ RAM).

If you need larger images at the moment then your best bet is to take the final output and run it through an ML upscaler. I'm using Topaz Gigapixel AI and it works really well, although it does highlight some of the imperfections in the images (which tend to be more hidden at smaller pixel sizes). I've tried running a couple of images through Gigapixel twice and got good quality up to about 25,000 pixels across, but at that point you do need to do some manual tidying up.

**Comment by ee0pdt** [comment]
What @fingermouse says

But for a free upscaler try: https://replicate.com/nightmareai/real-esrgan

**Comment by moonpost** [comment]
love the Diffusion Bee app versus the command line - would really love seeing Real ESRGAN built into it!

**Comment by fingermouse** [comment]
> What @fingermouse says
> But for a free upscaler try: https://replicate.com/nightmareai/real-esrgan

Sorry, yes — I should have mentioned that Gigapixel isn't free (although worth waiting for a sale on the bundle with Sharpen and Denoise if you're into photography). There are lots of ML upscalers out there, though, so you have plenty of options and as the models improve, they're getting really good. You can get some amazing results with a bit of time and practice.

**Comment by TonalS** [comment]
Stable Diffusion works well for 1024x1024 - the images are much better than what you get with an AI upscaler. Presumably it is just an extra option to add to the drop down list we already have

**Comment by tandefelt** [comment]
1024x1024 or higher, definitely would be amazing.. absolutely.. :), built in upscaler, ESRGAN etc would also be double amazing.. definitely. I use Topaz tools, which are also quite advanced..

**Comment by bennyblancodisco** [comment]
Please Please Please add in the 1024x1024 (or more) upscaling as part of Diffusion Bee.

**Comment by barryanders** [comment]
It's understandable why there would be size limits, however, I would second that going up to 1024 seems pretty harmless and would give the user the option to try it at least. I've gotten plenty of good results at that size.

**Comment by tandefelt** [comment]
Yes. Definitely, working with 1024x1024 from the server (SD), upscaling with Topaz AI tools starts to bring things into a whole another level.. So, upscaling overall is a central thing. that is the reason I suggested separately also consideration of LAOIN high resolution database.. fully understanding that the render times/processing etc can go way up. Still, results could be otherwordly. Topaz upscaled images already show promise, majorly.. but, being able to zoom in and see details would be.. amazing.

**Comment by impallari** [comment]
1024 pretty please with sugar on top! 🍒 🍒 🍒

**Comment by kengovaert** [comment]
> love the Diffusion Bee app versus the command line - would really love seeing Real ESRGAN built into it!

@moonpost it is available and working in my tests on version 2 https://github.com/divamgupta/diffusionbee-stable-diffusion-ui/releases/tag/2.2.1
